kyuubi-server/src/main/scala/org/apache/kyuubi/engine/flink/FlinkProcessBuilder.scala

```diff
@@ -80,13 +86,17 @@ class FlinkProcessBuilder(
     env.get("HADOOP_CONF_DIR").foreach(classpathEntries.add)
     env.get("YARN_CONF_DIR").foreach(classpathEntries.add)
     env.get("HBASE_CONF_DIR").foreach(classpathEntries.add)
-    val hadoopClasspath = env.get("HADOOP_CLASSPATH")
-    if (hadoopClasspath.isEmpty) {
-      throw KyuubiSQLException("HADOOP_CLASSPATH is not set! " +
-        "For more detail information on configuring HADOOP_CLASSPATH" +
-        "https://kyuubi.apache.org/docs/latest/deployment/settings.html#environments")
+    val hadoopCp = env.get(FLINK_HADOOP_CLASSPATH)
+    hadoopCp.foreach(classpathEntries.add)
+    val extraCp = conf.get(ENGINE_FLINK_EXTRA_CLASSPATH)
+    extraCp.foreach(classpathEntries.add)
+    if (hadoopCp.isEmpty && extraCp.isEmpty) {
+      throw new KyuubiException(s"The conf of ${FLINK_HADOOP_CLASSPATH} or " +
+        s"${ENGINE_FLINK_EXTRA_CLASSPATH} is empty." +
+        s"Please set ${FLINK_HADOOP_CLASSPATH} or ${ENGINE_FLINK_EXTRA_CLASSPATH} for " +
```

Comment:
the conf should be conf.key, for example, FLINK_HADOOP_CLASSPATH should be FLINK_HADOOP_CLASSPATH.key

Reply:
I didn't understand what you meant, this FLINK_HADOOP_CLASSPATH is a String, refer to HIVE_HADOOP_CLASSPATH in HiveProcessBuilder

Reply:
sorry, should be the conf of ENGINE_FLINK_EXTRA_CLASSPATH.

Reply:
ok I will change it